docs/source/AdministratorGuide/Tutorials/installWMS.rst

```diff
@@ -0,0 +1,340 @@
+  service WorkloadManagement_Matcher is installed, runit status: Run
+  [dirac-tuto]> install service WorkloadManagement OptimizationMind
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/OptimizationMind
+  service WorkloadManagement_OptimizationMind is installed, runit status: Run
+  [dirac-tuto]> install service WorkloadManagement SandboxStore
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/SandboxStore
+  service WorkloadManagement_SandboxStore is installed, runit status: Run
+  [dirac-tuto]> install service WorkloadManagement WMSAdministrator
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/WMSAdministrator
+  service WorkloadManagement_WMSAdministrator is installed, runit status: Run
+  [dirac-tuto]> install service Framework BundleDelivery
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/Framework/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/BundleDelivery
+  service WorkloadManagement_BundleDelivery is installed, runit status: Run
+  [dirac-tuto]> install service Framework Monitoring
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/Framework/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/Monitoring
+  service WorkloadManagement_BundleDelivery is installed, runit status: Run
+  [dirac-tuto]> install agent WorkloadManagement SiteDirector
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/SiteDirector
+  agent WorkloadManagement_SiteDirector is installed, runit status: Run
+  [dirac-tuto]> install agent WorkloadManagement JobCleaningAgent
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/JobCleaningAgent
+  agent WorkloadManagement_JobCleaningAgent is installed, runit status: Run
+  [dirac-tuto]> install agent WorkloadManagement PilotStatusAgent
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/PilotStatusAgent
+  agent WorkloadManagement_PilotStatusAgent is installed, runit status: Run
+  [dirac-tuto]> install agent WorkloadManagement StalledJobAgent
+  Loading configuration template /home/diracuser/DiracInstallation/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg
+  Adding to CS service WorkloadManagement/StalledJobAgent
+  agent WorkloadManagement_StalledJobAgent is installed, runit status: Run
+  [dirac-tuto]> install executor WorkloadManagement JobPath
```

Comment:
This isn't how the executors are installed in certification (and LHCbDirac?), where there is just one executor called "Optimizers" that calls the executors defined in the "Load" parameter.

Not sure what or if anything is better?

Reply:
You probably want one "optimizer" running all the executors.

Reply:
Alright, thanks I didn't know about that